assets/icons/progress-icon.js

```diff
@@ -0,0 +1,28 @@
+import { Rect, Path, SVG } from '@wordpress/components';
+
+export const ProgressIcon = () => (
+	<SVG
+		width="24"
+		height="24"
```

Comment:
Just the `viewBox` attribute should be enough.

Reply:
Fixed in 086583ec7b0cfb53b9df91023c109338c9ba14f3